monkey/monkey_island/cc/ui/src/components/report-components/SecurityReport.js

```diff
@@ -353,6 +364,29 @@ class ReportPageComponent extends AuthComponent {
     );
   }
 
+  generateZeroLogonOverview() {
+    let zeroLogonOverview = [];
+
+    // TODO finish this by linking to the documentation
+    if(!this.state.report.overview.issues[this.Issue.ZEROLOGON_PASSWORD_RESTORED]) {
+      zeroLogonOverview.push(<span>
+        <WarningIcon/> Automatic password restoration on a domain controller failed!
```

Comment:
We should add `<div className={'info-pane-warning'} key={'warning'}>` here so it's aligned properly, or remove it from [here](https://github.com/guardicore/monkey/pull/998/files?file-filters%5B%5D=.js&file-filters%5B%5D=.py#diff-59135095fe3ae314f1f3728eb08a85f1495f2e6a43c6797cf15ec4e28c76c9caR943) to maintain consistency

Reply:
Overviews and recommendations have different styles, which should be consistent with other overviews/recommendations. I improved the style of warning icon, but to add margins to this specific issue overview whilst other overviews don't have them doesn't make sense.